### DISCUSSION TITLE
Problem mit dem Einrichten von GitHub Actions für die automatische Bereitstellung

### DISCUSSION THREAD

**Discussion by Gulcan82** [community/community#144718] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hallo zusammen,

ich habe Schwierigkeiten beim Einrichten einer GitHub Actions-Workflow-Datei für die automatische Bereitstellung meines Projekts. Der Workflow soll das Backend auf Google Kubernetes und das Frontend auf Google Cloud deployen. Hier sind die Schritte, die ich bisher unternommen habe:
	1.	Ich habe die YAML-Dateien für das Deployment erstellt.
	2.	Das Docker-Image wurde erfolgreich gebaut und in Google Container Registry hochgeladen.
	3.	Bei der Ausführung des Workflows erhalte ich jedoch einen Fehler, und der Deployment-Prozess bricht ab.

Hat jemand bereits Erfahrung mit diesem Setup und könnte mir weiterhelfen? Ich wäre dankbar für jeden Hinweis oder eine Beispielkonfiguration, die in einer ähnlichen Umgebung funktioniert.

Vielen Dank im Voraus!

**Comment by shinybrightstar** [comment]
Hi @Gulcan82, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
